src/styles/ThoughtHeart.css

```diff
@@ -0,0 +1,22 @@
+.heart-container {
+  display: flex;
+  align-items: center;
+}
+
+.heart-count {
+  margin-left: 5px;
+}
+
+span {
+  background-color: #EAEAEA;
+  border-radius: 50%;
+  padding: 10px;
+  margin-right: 5px;
+}
+
+span.liked {
+  background-color: #FFADAD;
+  border-radius: 50%;
+  padding: 10px;
+  margin-right: 5px;
+}
```

Comment:
I think the heart-count style I mentioned in the ThoughtHeart.jsx file can be used here, and you can replace span with .heart-count. Using a specific class that you already created rather than the generic span selector can improve maintainability and clarity of your code. Something like this maybe ? 

```
.heart-count {
  background-color: #EAEAEA;
  border-radius: 50%;
  padding: 10px;
  margin-right: 5px;
/* Add cursor: pointer here */
  cursor: pointer; 
}

.heart-count.liked {
  /* Add styles for the liked state */
}
```